modules/vba_documents/app/serializers/vba_documents/upload_serializer.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+require_dependency 'common/exceptions'
+
+module VBADocuments
+  class UploadSerializer < ActiveModel::Serializer
+    type 'document_upload'
+
+    attributes :guid, :status, :location
+
+    def id
+      object.guid
+    end
+
+    delegate :status, to: :object
+
+    def location
```

Comment:
I set this up as two schemas in the swagger docs `DocumentUploadSubmission` and `DocumentUploadStatus`. Could reduce confusion about when the `location` was available, but probably premature.